### ISSUE TITLE
After upgrade to TFT43_V3.0.27 firmware, printer does not respond to touch command

### ISSUE THREAD

**Issue by twinsenfan** [bigtreetech/BIGTREETECH-TouchScreenFirmware#1840]
**Your question or details you need. Please describe.**
Where can I find the old TFT43_V3.0.26 firmware ? 

**Specify the device variant related to your question.**
TFT43_V3.0 updated to TFT43_V3.0.27 version
I had TFT43_V3.0.26 before the update, and it was running just fine. I am trying to downgrade before doing any further troubleshooting to isolate if the issue is caused by the new firmware.

**Additional context**
Add any other context or screenshots about the question here.

**Comment by kisslorand** [comment]
Put an empty `reset.txt` on you SDCard and reboot TFT. This will recalibrate the touch on your TFT.

**Comment by twinsenfan** [comment]
Thank you for the prompt response. Yes, I have done that, touched the red dot etc. The TFT will respond to the touch actions and I can switch between different menus. It also shows the current temperature of the hotend, bed and printer is ready status. Just non of the command(movement, heat up hotend seem to do anything)

**Comment by oldman4U** [comment]
Click. Commits to get access to all "versions" which have been released so far.
![Screenshot 2021-04-14 at 09 46 53](https://user-images.githubusercontent.com/54359396/114673467-83b56580-9d06-11eb-9885-2cbc556e512c.png)

**Comment by twinsenfan** [comment]
Didn't have luck finding the bin file for TFT43, is it normal that the bin for TFT43 screen is missing ?
![tft43](https://user-images.githubusercontent.com/7404094/114674656-6dc59600-9d42-11eb-9863-6f8d90fd6448.PNG)

**Comment by oldman4U** [comment]
For a version which has been released before the hardware was available - yes.

I just tested v27 on my TFT43 and it works. Given the fact that the update alone did not work properly, I would expect a problem with the SD card or the downloaded file.

Clicking on the "868 commits" I get this page:
![Screenshot 2021-04-14 at 10 43 50](https://user-images.githubusercontent.com/54359396/114681425-6e443980-9d0e-11eb-9587-9c2a97abeac3.png)
